src/app/data-model-provider/Provider.h

```diff
@@ -79,6 +79,14 @@ class Provider : public ProviderMetadataTree
     ///    - validation of ACL/timed interaction flags/writability, if those checks are desired.
     virtual ActionReturnStatus WriteAttribute(const WriteAttributeRequest & request, AttributeValueDecoder & decoder) = 0;
 
+    ///   Indicates the start/end of a series of list operations. This function will be called either before the first
+    ///   Write operation or after the last one of a series of consequence attribute data of the same attribute.
```

Comment:
"consequence" seems like the wrong word here.

Reply:
Yeah, I wasn't sure about that word either but since it was in the previous documentation from AAI I just took it from there. I think that rewriting the documentation will be better.